tests/acceptance/ember-advanced-tutorial-test.js

```diff
@@ -11,6 +11,18 @@ module('Acceptance | ember advanced tutorial', function (hooks) {
   const root = '/';
   const about = '/about';
   const contact = '/getting-in-touch';
+  const appPaths = [root, about, contact];
+
+  test.each(
+    'verify each route has the top nav',
+    appPaths,
+    async (assert, path) => {
+      await visit(path);
+      assert.strictEqual(currentURL(), path);
+      assert.dom('nav').exists();
+      assert.dom('h1').hasText('SuperRentals');
+    }
+  );
```

Comment:
The tutorial wanted me to write repeat test code and I just could not do it. After being thwarted several ways to Sunday trying to call an external fn, I finally stumbled upon this in the Qunit doc 🤘:
https://api.qunitjs.com/QUnit/test.each/



I was basically trying to avoid repeating this same assertion 3 times for each view and QUnit does not like you passing around `assert` (_I'm sure for a good reason, but the error is kind of vague..._):

```
const assertTopNavExists = () => {
  assert.dom('nav').exists();
  assert.dom('h1').hasText('SuperRentals');  
}

test('my test', async function (assert) {
  assertTopNavExists(assert);
})

```